src/examples/ExamplesPanel.ts

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { ensureDir, readFile, readJSON, writeJSON } from "fs-extra";
+import { ensureDir, readFile, readJSON, writeJSON, copy } from "fs-extra";
```

Comment:
I'm not sure why ExamplesPanel.ts is modified in this PR, I can't see not relation with rest of code.